test/cards/test_joker.py

```diff
@@ -560,6 +576,136 @@ def test_even_steven() -> None:
             assert j.get_mult_card(_make_card(rank=Rank.from_int(rank)), Mock(), Mock(jokers=[])) == 0
 
 
+@pytest.mark.unit
+def test_odd_todd() -> None:
+    trigger_ranks = [Rank.ACE, Rank.NINE, Rank.SEVEN, Rank.FIVE, Rank.THREE]
+    j = OddTodd()
+    for rank in range(1, 14):
+        if Rank.from_int(rank) in trigger_ranks:
+            assert j.get_chips_card(_make_card(rank=Rank.from_int(rank)), Mock(), Mock(jokers=[])) == 31
+        else:
+            assert j.get_chips_card(_make_card(rank=Rank.from_int(rank)), Mock(), Mock(jokers=[])) == 0
+
+
+@pytest.mark.unit
+def test_scholar() -> None:
+    j = Scholar()
+    for rank in range(1, 14):
+        card = _make_card(rank=Rank.from_int(rank))
+        chips = j.get_chips_card(card, Mock(), Mock(jokers=[]))
+        mult = j.get_mult_card(card, Mock(), Mock(jokers=[]))
+        if Rank.from_int(rank) == Rank.ACE:
+            assert chips == 20
+            assert mult == 4
+        else:
+            assert chips == 0
+            assert mult == 0
+
+
+@pytest.mark.unit
+def test_business_card() -> None:
+    j = BusinessCard()
+    face_ranks = [Rank.JACK, Rank.QUEEN, Rank.KING]
+    # Patch random.random to control output
+    with patch("random.random", return_value=0.4):
```

Comment:
Nit: tweak to make checked probability more self documenting
```suggestion
    with patch("random.random", return_value=0.0):
```